clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/v2/description/manifest/KubernetesManifestAnnotater.java

```diff
@@ -209,6 +209,26 @@ public static KubernetesManifestTraffic getTraffic(KubernetesManifest manifest)
     return new KubernetesManifestTraffic(loadBalancers);
   }
 
+  public static void setTraffic(KubernetesManifest manifest, KubernetesManifestTraffic traffic) {
+    Map<String, String> annotations = manifest.getAnnotations();
+    List<String> loadBalancers = traffic.getLoadBalancers();
+
+    if (annotations.containsKey(LOAD_BALANCERS)) {
+      KubernetesManifestTraffic currentTraffic = getTraffic(manifest);
+      if (currentTraffic.getLoadBalancers().equals(loadBalancers)) {
+        return;
+      } else {
+        throw new RuntimeException(String.format(
```

Comment:
Would this prevent adding a new load balancer to an existing (already load balanced) replica set? Not sure about the k8s best practices/constraints, just wondering what the implications are here.

Reply:
I think the idea was that folks should either 1) have all relevant load balancers stored as metadata on the manifest in source control or 2) have all relevant load balancers selected in the UI and added by Spinnaker to the manifest, since any combination of the two where Spinnaker adds to or overrides existing load balancers might be unexpected

Reply:
When we write up docs on this we should make this clear

Reply:
:+1:

Reply:
Good question.  We might also want to think about how this would interact with things like "find manifest" where a manifest already has another annotation...maybe at some point what you selected in the services section could just override the manifest if we need it for that feature.  But for the simple use cases I think this is simpler and we can revisit later.

Reply:
Sounds good.